airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -3,8 +3,9 @@ acceptance_tests:
   spec:
     tests:
       - spec_path: "integration_tests/spec.json"
+        # massively refactored `spec` 
         backward_compatibility_tests_config:
-          disable_for_version: "1.1.0"
+          disable_for_version: "1.2.0"
```

Comment:
curious, which backwards compatibility tests failed here? this shouldn't be noted as breaking